src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/InfrastructureFormula.kt

```diff
@@ -108,7 +122,7 @@ class InfrastructureFormula<out T : VirtualUsers>(
 
     private fun overrideJiraNetwork(
         network: Network
-    ): JiraFormula = when (jiraFormula) {
+    ): JiraFormula? = when (jiraFormula) {
```

Comment:
Why you added `?`. Is it possible to avoid that change?

Reply:
Because jira formula might be null. I wan't it to be avoided because there are two options to deal with jira (by formula or by already provisioned stuff)